### PR TITLE
[Update] Enable IP Sharing (Elastic IPs) in Atlanta through FRR

### PR DESCRIPTION
**Pull request by wildmanonline**

### CODE REVIEW

**Review by andystevensname — APPROVED** [review]

File: docs/guides/networking/linode-network/ip-sharing-atlanta/index.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+1.  Apply the `eth0` network interface configuration:
+
+    **Debian 10, Ubuntu 18.04 & CentOS 8**
+
+        sudo ifdown eth0 && sudo ifup eth0
+    **Ubuntu 20.04**
+
+        systemctl restart systemd-networkd
+
```
Comment (nmelehan) [inline docs/guides/networking/linode-network/ip-sharing-atlanta/index.md:235]: @wildmanonline should we include instructions for applying the network config with CentOS too?

File: docs/guides/networking/linode-network/ip-sharing-atlanta/index.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+1. [Disable Network Helper](/docs/platform/network-helper/#single-per-linode) on the Elastic IP Linodes and reboot them.
+
+## Install FRR
+
+This section provides instructions for installing FRR on Debian, Ubuntu, and CentOS systems through their native package managers. If you're using a different distribution or prefer to install FRR from source, follow [FRR's official installation instructions](http://docs.frrouting.org/en/latest/installation.html) to install FRR using git.
+
+### Ubuntu and Debian
+
+**Supported distributions:** *Ubuntu 20.04, 18.04, and 16.04 | Debian 11, 10, and 9*
```
Comment (nmelehan) [inline docs/guides/networking/linode-network/ip-sharing-atlanta/index.md:46]: @wildmanonline not all of these support distributions are given instructions in the Configure FRR section (Ubuntu 16.04, Debian 11 and 9). Should we consider pruning the list of supported distributions here? Or tell the reader that configuration instructions for those distros are outside the scope of the guide?

File: docs/guides/networking/linode-network/ip-sharing-atlanta/index.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+        echo deb https://deb.frrouting.org/frr $(lsb_release -s -c) $FRRVER | sudo tee -a /etc/apt/sources.list.d/frr.list
+
+1.  Install the FRR package:
+
+        sudo apt update && sudo apt install frr frr-pythontools
+
+### CentOS/RHEL
+
+**Supported distributions:** *CentOS Stream (and 8), CentOS 7, other RHEL derivatives (including AlmaLinux 8, and Rocky Linux 8), and Fedora.*
```
Comment (nmelehan) [inline docs/guides/networking/linode-network/ip-sharing-atlanta/index.md:70]: @wildmanonline not all of these support distributions are given instructions in the Configure FRR section (CentoS Stream, 7, and all the RHEL derivatives + Fedora). Should we consider pruning the list of supported distributions here? Or tell the reader that configuration instructions for those distros are outside the scope of the guide?

Comment (wildmanonline) [reply docs/guides/networking/linode-network/ip-sharing-atlanta/index.md:70]: I updated the distros under the Configure FRR section to include all supported distros.

File: docs/guides/networking/linode-network/ip-sharing-atlanta/index.md (hunk 1 above)
Comment (wildmanonline) [reply docs/guides/networking/linode-network/ip-sharing-atlanta/index.md:236]: I think those are handled with `sudo ifdown eth0 && sudo ifup eth0` in the section above that includes CentOS. I renamed that section's bold psuedo-header to "Debian, Ubuntu (18.04 and 16.04), and CentOS/RHEL"

**Review by nmelehan — APPROVED** [review]
